services/repository/transfer.go

```diff
@@ -41,7 +41,7 @@ func TransferOwnership(doer, newOwner *models.User, repo *models.Repository, tea
 	}
 
 	for _, team := range teams {
-		if err := team.AddRepository(newRepo); err != nil {
+		if err = team.AddRepository(newRepo); err != nil {
```

Comment:
Put the assignment back in - you're hurting the compiler's ability to use a throwaway assignment here.

```suggestion
		if err := team.AddRepository(newRepo); err != nil {
```